### DISCUSSION TITLE
NumFOCUS Affiliated Projects application

### DISCUSSION THREAD

**Discussion by rht** [mesa/mesa#1096] [General]
According to https://numfocus.org/sponsored-projects/affiliated-projects, the next deadline is on Jan 15 2022.

**Comment by tpike3** [comment]
@rht Very cool!

@jackiekazil @dmasad It seems the big question is whether to be [fiscally sponsored or affiliated]( https://numfocus.org/projects-overview).

**Reply by rht** [reply]
I think it is safe to start with being affiliated, and then eventually upgrade to be fiscally sponsored.

**Reply by jackiekazil** [reply]
Fiscally is if we ever went after money -- eg grants. Then we need more of a fiduciary component. 
@rht is correct -- we want affiliated. We should discuss during our next dev meeting.

**Reply by tpike3** [reply]
I realized we had missed the deadline, but hastily through it together. Fingers crossed they still consider it.
